src/parseSetupScript.ts

```diff
@@ -0,0 +1,41 @@
+// import ts from 'typescript';
+
+export default async function parseSetupScript(script: HTMLScriptElement, scriptCode: string) {
+    const isTypeScript = (script?.attributes as any)?.lang?.value === 'ts'
+    const sc = document.createElement('script')
+    sc.src = 'https://unpkg.com/typescript/lib/typescript.js'
+    const promise = new Promise((resolve, reject) => {
+        sc.onload = resolve
+        sc.onerror = reject
+    })
+    document.head.appendChild(sc)
+    await promise
+    const ts = (window as any).ts
+    const sf = ts.createSourceFile(
+        "test." + (isTypeScript ? "ts" : "js"),
+        scriptCode,
+        99, // ts.ScriptTarget.LATEST,
```

Comment:
why not just use `ts.ScriptTarget.LATEST` instead of 99 ?

Reply:
These weren't available in this typescript lib build